pkg/packetfilter/nftables/nftables.go

```diff
@@ -79,7 +80,11 @@ type packetFilter struct {
 	nftables knftables.Interface
 }
 
-func New() (packetfilter.Driver, error) {
+func New(family k8snet.IPFamily) (packetfilter.Driver, error) {
+	if family != k8snet.IPv4 {
```

Comment:
Maybe we can also enable  IPV6 driver creation for nftables, it should be safe because currently all clients will only create a V4 driver